### ISSUE TITLE
Plastic mixer CT method extension

### ISSUE THREAD

**Issue by wormzjl** [TeamPneumatic/pnc-repressurized#429]
### Minecraft Version
1.12.2
### Forge Version
2847
### Mod Version
pneumaticcraft-repressurized-1.12.2-0.11.10-385
### Describe your problem, including steps to reproduce it
I have the following script
`mods.pneumaticcraft.plasticmixer.addSolidifyOnlyRecipe(<liquid:rubber> * 100, <ic2:crafting:0>);`
Which is intended to make IC2 rubber (meta=0) from liquid rubber
However, when looking at recipes ingame, this rubber liquid can make everything else having the same item ID but different meta
![image](https://user-images.githubusercontent.com/2989009/75902615-93465780-5e40-11ea-8a65-c4e0c59ca20a.png)

Also, it seems that with custom recipes it enforces the consumption of all 3 dies, can you make it possible to specify this in the CT script (or just disable the consumption)? Thanks!

**Comment by desht** [comment]
Not really a bug: the plastic mixer was originally only ever intended to solidify plastic, but the extension of functionality was meant to add solidifying & dyeing to other colourable blocks like glass or concrete)

However, the functionality you're looking for (configurable dye use & limiting to specific item meta) shouldn't be hard to add, so I'll treat this as a feature request.

**Comment by wormzjl** [comment]
Ok thanks!

**Comment by desht** [comment]
I've made a change and you can test at it https://jenkins.k-4u.nl/job/PneumaticCraft-Repressurized/, build 387+

Try the following CT syntax:
```
mods.pneumaticcraft.plasticmixer.addSolidifyOnlyRecipe(<liquid:rubber> * 100, <ic2:crafting>, false, 0);
```
The `false` parameter prevents dye usage, and the 0 is the specific item meta to use.

**Comment by wormzjl** [comment]
Great, seems to be working!

**Comment by desht** [comment]
Fixed in 0.11.11 release
